_data/blogs.yml

```diff
@@ -273,6 +273,11 @@
   img: haider2.jpg
   author_name: Haider Shafaq
 
+- url: https://workingwithfossasiawordpress.wordpress.com/2018/01/06/participation-in-open-source-community/
+  title: Participation in Open source community
+  img: Shahaam.jpeg
```

Comment:
The image that you uploaded and wrote here don't have the same name.
One ends with jpg and the other with jpeg.
Please make it same